viewer/test/viewer-test-pptr.js

```diff
@@ -262,7 +281,7 @@ describe('Lighthouse Viewer', () => {
 
       const savedPage = await browser.newPage();
       const savedPageErrors = [];
-      savedPage.on('pageerror', pageError => savedPageErrors.push(pageError));
+      savedPage.on('pageerror', e => savedPageErrors.push(`${e.message} ${e.stack}`));
```

Comment:
nit: I thought that `e.stack` would also include `e.message` by default. Is that no the case here?